test/fixtures/manifest/config/config-with-remote-extends.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "owner/repo/main",
```

Comment:
Perhaps:
* `owner/repo@branchOrSha:path/to/file.json`
* `owner/repo/path/to/file.json@branchOrSha`
* `owner/repo:path/to/file.json@branchOrSha`

Or something similar to https://docs.renovatebot.com/config-presets/#github

Reply:
We can combine. Added support for _renovate-like_ format and for the last two options.